spec/models/user_stat_spec.rb

```diff
@@ -72,6 +72,35 @@
         end
       end
     end
+
+    context 'post_edits_count' do
+      let!(:post) { Fabricate(:post) }
+      let(:update_post!) { Fabricate(:post_revision, user: user, post: post) }
```

Comment:
```suggestion
      fab!(:post_revision) { Fabricate(:post_revision, user: user) }
      fab!(:post) { post_revision.post }
```

At Discourse, we prefer to use [fab!](https://github.com/discourse/discourse/blob/0e4b8c5318569ef7e7a111563709699e3b9ce219/spec/rails_helper.rb#L159-L161) to prefabricate objects for performance reasons.